ng2-components/ng2-alfresco-documentlist/src/components/document-list.component.ts

```diff
@@ -513,6 +515,29 @@ export class DocumentListComponent implements OnInit, OnChanges, AfterContentIni
         });
     }
 
+    private loadMemberSites(): void {
+        const options = {
+            include: [ 'properties' ],
+            maxItems: this.pageSize,
+            skipCount: this.skipCount
+        };
+
+        // TODO: Change `result` type once ADF-1829 gets sorted out
```

Comment:
Can you add the right type and remove the comment I am going to  do the fix on the JS-API

Reply:
the right one is `SitePaging`  as per docs, right?

Reply:
https://github.com/Alfresco/alfresco-js-api/pull/287